docs/components/datetime-picker.md

```diff
@@ -0,0 +1,453 @@
+        { key: "24h", prefix: "Last", timeframeText: "24 hours", start: function() {}, end: function() {}},
+        ...
+      ]
+    },
+    {
+      section: "Current",
+      values: [
+        { key: "current_week", prefix: "Current", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "current_month", prefix: "Current", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+    {
+      section: "Previous",
+      values: [
+        { key: "previous_week", prefix: "Previous", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "previous_month", prefix: "Previous", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+  ]
+/>
+```
+
+### Relative time frames only
+
+Set the `v-model` to [Range date time picker](#range-date-time-picker-v-model)
+This utilizes the same time frames as the previous example; however, in this example we are passing in a pre-selected time frame.
+
+<div>
+  <KDateTimePicker
+    v-model="currentValue6"
+    @change="newVal => emitVal6 = newVal"
+    placeholder="Please select a time frame"
+    mode="relative"
+    width="480"
+    :minDate="minDate"
+    :maxDate="maxDate"
+    :range="true"
+    :timePeriods="exampleTimeFrames"
+  />
+  <div class="mt-6">Emitted value: <pre class="json">{{ emitVal6 }}</pre></div>
+</div>
+
+```html
+<KDateTimePicker
+  v-model="currentValue"
+  @change="newVal => emitVal = newVal"
+  placeholder="Please select a time frame"
+  mode="relative"
+  width="480"
+  :minDate="minDate"
+  :maxDate="maxDate"
+  :range="true"
+  :timePeriods=[
+    {
+      section: "Last",
+      values: [
+        { key: "15m", prefix: "Last", timeframeText: "15 minutes", start: function() {}, end: function() {} },
+        { key: "12h", prefix: "Last", timeframeText: "12 hours", start: function() {}, end: function() {} },
+        { key: "24h", prefix: "Last", timeframeText: "24 hours", start: function() {}, end: function() {} },
+        ...
+      ]
+    },
+    {
+      section: "Current",
+      values: [
+        { key: "current_week", prefix: "Current", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "current_month", prefix: "Current", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+    {
+      section: "Previous",
+      values: [
+        { key: "previous_week", prefix: "Previous", timeframeText: "week", start: function() {}, end: function() {} },
+        { key: "previous_month", prefix: "Previous", timeframeText: "month", start: function() {}, end: function() {} }
+      ]
+    }
+  ]
+/>
+```
+
+## Props
+
+### v-model
+
+#### Single date time picker `v-model`
+
+A single date or time picker instance which can be seeded with a value like so:
+
+```js
+<KDateTimePicker
+  v-model="currentValue"
+  ...
+/>
+Where `currentValue` is a valid Date object, eg: `new Date()`
+```
```

Comment:
The formatting here is incorrect.

The code block should be of type `html` and then the "Where..." text just regular text. Something like this might be easier to read IMO.

Same comment applies below for a lot of the props sections.

```suggestion
A single date or time picker instance which can be seeded with a value as shown here, where `currentValue` is a valid Date object, eg: `new Date()`.

```html
<KDateTimePicker
  v-model="currentValue"
/>
```
```